plugins/modules/mso_schema_site_external_epg.py

```diff
@@ -144,11 +158,17 @@ def main():
     site = module.params.get("site")
     external_epg = module.params.get("external_epg")
     l3out = module.params.get("l3out")
+    l3out_schema = module.params.get("l3out_schema")
+    l3out_template = module.params.get("l3out_template")
     route_reachability = module.params.get("route_reachability")
     state = module.params.get("state")
 
     mso = MSOModule(module)
 
+    l3out_template = template if l3out_template is None else l3out_template.replace(" ", "")
```

Comment:
Why are we specifically removing spaces from the template name? Unless I am missing anything.

Reply:
Ah okay, good to know thanks.

Reply:
Then just a suggestion would be to document that behaviour in the description of l3out_template.